client/asset/interface.go

```diff
@@ -316,8 +316,13 @@ type ConfigOption struct {
 	Repeatable string `json:"repeatable"`
 	// RepeatN signals how many times text input should be repeated, replicating
 	// this option N times.
-	RepeatN  int32 `json:"repeatN"`
-	Required bool  `json:"required"`
+	RepeatN        int32 `json:"repeatN"`
+	Required       bool  `json:"required"`
+	QuoteAssetOnly bool  `json:"quoteAssetOnly"`
```

Comment:
I don't know if it's working

![image](https://github.com/decred/dcrdex/assets/6109680/d34acb92-5db5-4160-9134-33426721b2d9)

Reply:
"Base Asset Balance" is setting how much of the base asset balance this bot should be allocated. There is no "Multi Split Buffer" for the base asset. This section could definitely use some UI improvement though.

Reply:
Oh, right. Sorry.